boxes/boxes.py

```diff
@@ -149,10 +180,132 @@ def _get_box_hash(self, box_name: str) -> str:
         return stdout.decode('utf-8').strip().split()[0]
 
 
+class Builder:
+    IGNORED_PACKAGES = [
+        'debconf-2.0',
+    ]
+
+    def __init__(self, destination: pathlib.Path, extract: bool):
+        self._destination = destination
+        self._extract = extract
+        self._logger = logging.getLogger('builder')
+        self._workdir = pathlib.Path(
+            tempfile.mkdtemp(prefix='sio2jail-box')
+        )
+        self._root = self._workdir / 'root'
+        self._root.mkdir()
+        self._cmd = boxes.utils.cmd.CommandRunner(self._logger)
+
+    def __del__(self) -> None:
+        try:
+            with self._cmd.sudo():
+                self._cmd.run('chmod', '-R', 'a+rwX', str(self._workdir))
+            self._logger.info("Deleting workdir %s", self._workdir)
+            shutil.rmtree(str(self._workdir))
+        except Exception:
+            self._logger.exception("Failed to delete workdir %s", self._workdir)
+
+    def run(self, boxes_names: List[str]):
+        self._prepare_chroot()
+        for box in boxes_names:
+            self._build_box(box)
+
+    def _prepare_chroot(self) -> None:
+        with self._cmd.sudo():
+            self._cmd.run('debootstrap', 'stable', str(self._root))
```

Comment:
Po 1. to nie powinniśmy używać `stable`, bo to może znaczyć różne rzeczy w różnych momentach.
Po 2. chcemy móc jakoś przekazać nazwę wersji z zewnątrz.

Reply:
Aaaaaaaa wiem, wiem, to dalej ma WIP w nazwie ;p